packages/react-core/src/components/HelperText/HelperTextItem.tsx

```diff
@@ -6,41 +6,42 @@ import ExclamationTriangleIcon from '@patternfly/react-icons/dist/esm/icons/excl
 import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
 import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
 
+export enum HelperTextItemVariant {
+  default = 'default',
+  warning = 'warning',
+  error = 'error',
+  success = 'success'
+}
```

Comment:
Though how each component uses these variants can be slightly different. Here it's these 4 variants, Alert has `danger | success | warning | info | custom`, Progress only has `dange | success | warning`, etc. So it could also be a reason to make components follow a more universal variant naming when we can. At the very least, re-evaluating whether "error" should be used instead of danger in some places or vice versa, and whether `___Variant` or `___Status` makes more sense as an enum/prop name.